internal/driver/glfw/window.go

```diff
@@ -1533,3 +1535,17 @@ func isKeyModifier(keyName fyne.KeyName) bool {
 		keyName == desktop.KeyAltLeft || keyName == desktop.KeyAltRight ||
 		keyName == desktop.KeySuperLeft || keyName == desktop.KeySuperRight
 }
+
+//lint:ignore U1000 This is fine, just a helper function for some platforms
```

Comment:
I'm wondering if there perhaps would be a better way to do this, without needing the ignore.

Reply:
Yes, the other possibility was to create a new file for this function. I have done that now